packages/storybook/config/storybook/story.config.js

```diff
@@ -4,7 +4,7 @@ let outputChangedComponentPackages;
 
 const getChangedPackageStories = () => {
     try {
-        outputChangedComponentPackages = execSync('npx turbo run build --filter=[origin/master] --dry=json --filter=!@justeat/f-cookie-banner-static', {
+        outputChangedComponentPackages = execSync('npx turbo run build --filter=...[origin/master] --dry=json --filter=!@justeat/f-cookie-banner-static', {
```

Comment:
Return a JSON object of changed components and their dependants to be served by Storybook.